packages/run-it/src/components/RequestForm/RequestForm.spec.tsx

```diff
@@ -47,6 +47,48 @@ describe('RequestForm', () => {
     jest.resetAllMocks()
   })
 
+  describe('validation messages', () => {
+    test('validation errors are displayed', () => {
+      const message = 'Invalid message'
+      renderWithTheme(
+        <RequestForm
+          configurator={defaultConfigurator}
+          setVersionsUrl={runItNoSet}
+          inputs={[
+            {
+              name: 'user_id',
+              location: 'path',
+              type: 'string',
+              required: true,
+              description: 'A unique identifier for a user',
+            },
+          ]}
+          handleSubmit={handleSubmit}
+          httpMethod={'GET'}
+          requestContent={requestContent}
+          setRequestContent={setRequestContent}
+          needsAuth={false}
+          hasConfig={true}
+          sdk={mockSdk}
+          setHasConfig={() => true}
+          isExtension={false}
+          validationMessage={message}
+          handleConfig={runItNoSet}
+        />
+      )
+
+      expect(screen.getByRole('img', { name: 'Error' })).toBeInTheDocument()
+      expect(screen.getByText(message)).toBeInTheDocument()
+    })
+
+    test.skip('clear removes validation messages', () => {
+      // TODO implement the clear hook
+    })
+
+    test.skip('clicking run with an invalid body shows a messagebar', () => {
+      // TODO implement the run hook
```

Comment:
Are these intentionally skipped? I think they should be part of this PR

Reply:
Yes, I marked them as `test.todo`